src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/metrics/KustoKafkaMetricsUtil.java

```diff
@@ -0,0 +1,54 @@
+package com.microsoft.azure.kusto.kafka.connect.sink.metrics;
```

Comment:
since this is util class please do add a private constructor to prevent instantiation as mentioned below
 
  private KustoKafkaMetricsUtil() {
    throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
  }